src/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Footer = (props) => {
```

Comment:
Can we put this in it's own component directory or group directory for common components?

https://reactjs.org/docs/faq-structure.html#grouping-by-features-or-routes